packages/webapp/src/components/Preview/Heading/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import calcPrettyName from './calcPrettyName';
+
+import CustomizedDateSelect from './CustomizedDateSelect';
+import CustomizedSelect from './CustomizedSelect';
+
+
+import {
+  HeadingWrapper,
+  HeadingContainer,
+  HeadingText,
+  Title,
+  SelectsGroup,
+  RightOptions,
+  Link,
+  ButtonDetails,
+  ButtonText,
+  ArrowStyled
+ } from './styled';
+
+
+const callDetailedAnalysisButton = url => (
+  <Link href={url}>
+    <ButtonDetails>
+      <ButtonText>Detailed Analysis</ButtonText>
+      <ArrowStyled />
+    </ButtonDetails>
+  </Link>
+);
+
+const Heading = ({ government, departmentNames, selected, eventHandler, year, sphere }) => {
+  const url = `/${year}/${sphere}/${government === 'south-africa' ? '' : `${government}/`}departments/${selected}`;
```

Comment:
This is getting a bit hard to read, perhaps break it out into a function called `provinceFolder`.

Reply:
@ZeeshaanMaudar `const provinceFolder = government === 'south-africa' ? '' : \`${government}/\``

Reply:
Done and replaced with provinceFolder inside of url